docs/fundamentals/executor/hub/push-executor.md

```diff
@@ -16,7 +16,9 @@ jina hub push [--public/--private] <path_to_executor_folder>
 
 <script id="asciicast-tpvuZ9u0lU2IumRyLlly3JI93" src="https://asciinema.org/a/tpvuZ9u0lU2IumRyLlly3JI93.js" async></script>
 
-It will return `NAME` & `SECRET`, which you will need to use (if the Executor is private) or update the Executor. **Please keep them carefully.**
+If you logged in Jina: It will return `TASK_ID`, which you will need to use it to get building status of an executor. **Please keep them carefully.**
+
+If you not logged in Jina: It will return `NAME` & `SECRET`, which you will need to use (if the Executor is private) or update the Executor. **Please keep them carefully.**
```

Comment:
```suggestion
If you have logged in Jina, it will return `TASK_ID`. You need that to get build status and logs. 

If you haven't logged in Jina, it will return `NAME` and `SECRET`. You will need them to use (if the Executor is private) or update the Executor. **Please keep them carefully.**
```